Docs/sphinx_documentation/source/AMReX_Profiling_Tools.rst

```diff
@@ -287,10 +287,39 @@ macros in the following way:
               }
           }
 
+The ``MyFuncs`` region appears in the Tiny Profilier output as an additional table.
+The following output example, mimics the above code. In it, the region is
+indicated by ``REG::MyFuncs``.
+
+.. code-block:: console
+
+    BEGIN REGION MyFuncs
+
+    -------------------------------------------------------------
+    Name          NCalls  Excl. Min  Excl. Avg  Excl. Max   Max %
+    -------------------------------------------------------------
+    MyFunc0         1000      4.402      4.402      4.402  14.19%
+    MyFunc1         1000       4.39       4.39       4.39  14.15%
+    REG::MyFuncs    1000     0.0168     0.0168     0.0168   0.05%
+    -------------------------------------------------------------
+
+    -------------------------------------------------------------
+    Name          NCalls  Incl. Min  Incl. Avg  Incl. Max   Max %
+    -------------------------------------------------------------
+    REG::MyFuncs    1000      8.809      8.809      8.809  28.39%
+    MyFunc0         1000      4.402      4.402      4.402  14.19%
+    MyFunc1         1000       4.39       4.39       4.39  14.15%
+    -------------------------------------------------------------
+
+    END REGION MyFuncs
+
```

Comment:
Should this go down under the "Sample Output" section? Seems out of place here, given that section down below.

Reply:
Ah, I've been so focused on the named regions I missed that. 

Actually, how about we combine the Sample Output Section with the Tiny profile section?  Then the additional region sample would make more sense.